<?xml version="1.0"?>
<launch>
    <include file="$(find aruco_proxy_pose_estimation)/launch/apriltag_grid_detector.launch">
        <arg name="config_file" value="$(find corsmal_setup_bringup)/config/apriltag_grid_detector_config_left_azure.yaml"/>
    </include>
    <node pkg="corsmal_setup_bringup" type="calibrate_left_camera.py" name="calibrate_left_camera" output="screen"/>
</launch>